api/v1alpha2/virtualmachine_network_types.go

```diff
@@ -178,6 +178,27 @@ type VirtualMachineNetworkSpec struct {
 	// +optional
 	Disabled bool `json:"disabled,omitempty"`
 
+	// Nameservers is a list of IP4 and/or IP6 addresses used as DNS
+	// nameservers. These are applied globally.
+	//
+	// Please note this feature is available only with the following bootstrap
```

Comment:
I think we want to depart from the pattern here and state:

```go
// Please note global nameservers are only available with the following
// bootstrap providers: LinuxPrep and Sysprep. The Cloud-Init bootstrap
// provider supports per-interface nameservers.
```

Otherwise I am concerned users will not realize nameservers are supported at all with Cloud-Init.